src/Sustainsys.Saml2/AspNetCore/Saml2Extensions.cs

```diff
@@ -1,14 +1,14 @@
 ﻿using Microsoft.AspNetCore.Authentication;
+using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.DependencyInjection.Extensions;
 using Microsoft.Extensions.Options;
-using Sustainsys.Saml2.AspNetCore;
 using Sustainsys.Saml2.Bindings;
 using Sustainsys.Saml2.Serialization;
 
 
 // By convention, the extension methods are placed in the Microsoft shared 
 // namespace to allow convenient access from intellisense.
-namespace Microsoft.Extensions.DependencyInjection;
```

Comment:
The AddSaml2() methods should be in the `Microsoft.Extensions.DependcyInjection` namespace. This aligns with how the Microsoft supplied handlers work - they put their `Add*` methods in that namespace even though the rest of the code is in `Microsoft.AspNetCore.Authentication.*`